.svglintrc.js

```diff
@@ -473,6 +473,40 @@ module.exports = {
                 ignoreIcon(reporter.name, iconPath, $);
               }
             }
+          },
+          function(reporter, $, ast) {
+            reporter.name = "path-format";
+
+            const iconPath = $.find("path").attr("d");
+
+            const validPathFormatRegex = /^[Mm][MmZzLlHhVvCcSsQqTtAaEe0-9-,.\s]+$/;
+            if (!validPathFormatRegex.test(iconPath)) {
+              let errorMsg = "Invalid path format", reason;
+
+              if (!(/^[Mm]/.test(iconPath))) {
+                // doesn't start with moveto
+                reason = `should start with \"moveto\" command (\"M\" or \"m\"), but starts with \"${iconPath[0]}\"`;
+                reporter.error(`${errorMsg}: ${reason}`);
+              }
+
+              const validPathCharacters = "MmZzLlHhVvCcSsQqTtAaEe0123456789-,. ",
+                    invalidCharactersMsgs = [],
+                    pathDStart = '<path d="',
+                    pathDIndex = $.html().indexOf(pathDStart) + pathDStart.length;
+
+              for (let i=1; i<iconPath.length; i++) {
+                if (validPathCharacters.indexOf(iconPath[i]) === -1) {
+                  invalidCharactersMsgs.push(`"${iconPath[i]}" at index ${pathDIndex + i}`);
+                }
+              }
+
+              // contains invalid characters
+              if (invalidCharactersMsgs.length) {
```

Comment:
I personally prefer to be explicit, but it's not important

```suggestion
              if (invalidCharactersMsgs.length > 0) {
```